analyzers/tests/SonarAnalyzer.TestFramework/Verification/DiagnosticVerifier.cs

```diff
@@ -130,7 +130,7 @@ public static class DiagnosticVerifier
 
             ReportDiagnostic Severity(string id)
             {
-                if (id == AnalyzerFailedDiagnosticId)
+                if (id == AD0001)
                 {
                     return ReportDiagnostic.Error;
                 }
```

Comment:
CAYC: I think it would be nice to re-ident the next ternary for readability:

```csharp
                    return !onlyDiagnostics.Any() || onlyDiagnostics.Contains(id)
                        ? ReportDiagnostic.Warn
                        : ReportDiagnostic.Suppress;
```

Reply:
It's rewritten in #8673